### DISCUSSION TITLE
MDI deprecated brand icons

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#14715] [Ideas]
I noticed Material Design Icons library has deprecated all brand icons, and they are set to be removed in the v8 release.  Details here: https://pictogrammers.com/docs/library/mdi/guides/brand-icons/

This will affect most, if not all the icons used for authentication backends here: https://github.com/netbox-community/netbox/blob/c78a792cccfdbe6f373c0d474b1620e56e5f9cf8/netbox/netbox/authentication.py#L21

It sounds like their suggestion is to switch to [Simple Icons](https://github.com/sachinraja/simple-icons-py).

**Comment by abhi1693** [comment]
I'm not sure if this can be used but we host the biggest icon library out there with a whoping count of 70k

https://onemind-services-llc.github.io/react-icons-ng
